rosette/src/Import.cc

```diff
@@ -68,10 +75,15 @@ pOb createRosetteObject(ObjectCodePB::Object * ob) {
     }
 
     // If this is a reference, find the real object
-    if (ob->reference() != 0) {
-        ob = importObjectCode.mutable_objects(oi->pbIndex);
-        if (VerboseFlag) {
-            fprintf(stderr, " Referenced Object=\n%s\n", ob->DebugString().c_str());
+    if (ob->reference()) {
+        if (oi->pbIndex >= 0) {
+            ob = importObjectCode.mutable_objects(oi->pbIndex);
+            if (VerboseFlag) {
+                fprintf(stderr, " Referenced Object=\n%s\n", ob->DebugString().c_str());
```

Comment:
Don't you want a \n here like on line 59?

Reply:
The one on line 59 is to separate it from the function name. I can add it later if we want.